From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Karolina Surma <ksurma@redhat.com>
Date: Fri, 19 Jan 2024 12:00:18 +0100
Subject: [PATCH] 00418: Don't generate sbom in make regen-all

The script and make target, modified in Python 3.13.0a4,
assumes the presence of expat and _decimal/libmpdec libraries, resulting in an
error and failed build when not found.
Reported upstream: https://github.com/python/cpython/issues/115663
---
 Makefile.pre.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index 8252e6631c..3f4a89ad07 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -1625,10 +1625,11 @@ regen-unicodedata:
 regen-all: regen-cases regen-typeslots \
 	regen-token regen-ast regen-keyword regen-sre regen-frozen \
 	regen-pegen-metaparser regen-pegen regen-test-frozenmain \
-	regen-test-levenshtein regen-global-objects regen-sbom regen-jit
+	regen-test-levenshtein regen-global-objects regen-jit
 	@echo
 	@echo "Note: make regen-stdlib-module-names, make regen-limited-abi, "
-	@echo "make regen-configure and make regen-unicodedata should be run manually"
+	@echo "make regen-configure, make regen-unicodedata and make regen-sbom "
+	@echo "should be run manually"
 
 ############################################################################
 # Special rules for object files
